addons/payment/models/payment_provider.py

```diff
@@ -247,6 +247,20 @@ def _compute_feature_support_fields(self):
 
     #=== ONCHANGE METHODS ===#
 
+    @api.constrains('capture_manually')
+    def _check_capture_manually(self):
```

Comment:
This must also be checked when enabling a payment method after that manual capture is enabled on the provider.